### ISSUE TITLE
Reviving vscode-sftp

### ISSUE THREAD

**Issue by ghost** [liximomo/vscode-sftp#875]
The maintainer of this repository is no longer active on GitHub. Seeing that this extension is still very popular and mostly functional on VSCode, I am going to take up the reins of forking it and maintaining it myself.

Here is the fork: https://github.com/bthompson90/vscode-sftp-revived

I haven't released anything to the marketplace yet. I want to get it in a good working state first so that some of the bugs are fixed before releasing it to the wild, or else there is really no point.

It's licensed by MIT, so that's a big plus!

Cheers,
LT

**Comment by GusGF** [comment]
If people subscribe to this post will we be notified when something is released? 

Thank you for doing this :)

**Comment by ghost** [comment]
@GusGF yes, I will notify people in this thread once the initial release is out there.

**Comment by ghost** [comment]
Here is a list of issues that I'm going to be tackling in the first release: https://github.com/TomaisLeannan/vscode-sftp-revived/projects/1

**Comment by AlexMarcoDAngelo** [comment]
Maybe the very first thing to address is that the extension does not work in 1.53.x of VSCode..

**Comment by Stonerea656** [comment]
I love this extension!, except cant access Siteground, its fine on other servers? Filezilla requires key pairs to be setup initially with sftp  so does that mean there is a setup step missed with this extension?

**Comment by ghost** [comment]
> Maybe the very first thing to address is that the extension does not work in 1.53.x of VSCode..

Yes that does look like a good place to start.

**Comment by ghost** [comment]
> I love this extension!, except cant access Siteground, its fine on other servers? Filezilla requires key pairs to be setup initially with sftp  so does that mean there is a setup step missed with this extension?

Possibly. Im not familiar with Siteground. Do you mind opening up a new issue deacribing your problem?

**Comment by Craigeniah** [comment]
@thompsbrian thanks for taking the reigns here, I use this plugin everyday and have seen the need for it's reviving. Much appreciated!

**Comment by ghost** [comment]
Btw I'm accepting any PRs.  Need all the help I can get with a kid coming and stuff.

**Comment by oneilonline** [comment]
> Here is the fork: https://github.com/bthompson90/vscode-sftp-revived
> LT

Link is 404 broken...

Looks like this one: https://github.com/viperet/vscode-sftp-revived

**Comment by jeremy-farrance** [comment]
Yup, whoever "bthompson90" was, he must have got to busy and backed off completely and recently turned off his GitHub account or something.

**Comment by nickdekruijk** [comment]
Looking at all the forks of this repo this one seems updated most recently: https://github.com/Natizyskunk/vscode-sftp/
Anyone tried this yet?

**Comment by marcolaux** [comment]
> Looking at all the forks of this repo this one seems updated most recently: https://github.com/Natizyskunk/vscode-sftp/
> Anyone tried this yet?

~~does not work for me~~

ssh-streams module has to get installed manually for the [v.1.12.10 release](https://github.com/Natizyskunk/vscode-sftp/releases/tag/v1.12.10) - then it seems to work:
https://github.com/liximomo/vscode-sftp/issues/919#issuecomment-843392014

**Comment by cyberfish** [comment]
How can I install the extension from https://github.com/viperet/vscode-sftp-revived? Because, in VS-Code is not this Extension. Thanks.

**Comment by kylewetton** [comment]
@cyberfish clone the repo to your computer, install `vsce` on your machine with `npm install -g vsce`

Now open the repo in VSCode and in the root directory run `vsce package`

This creates a `.vsix` file. This file is an extension file. You can install extensions through the Marketplace, but you can also install extensions directly with a .vsix file.

Have a look here: 

<img width="480" alt="Screen Shot 2021-06-14 at 7 59 26 PM" src="https://user-images.githubusercontent.com/40445966/121858768-0da39e00-cd4b-11eb-8904-6b90a25cf3f8.png">

**Comment by Natizyskunk** [comment]
You can try my version, i've fixed a lot of bugs, maybe this will fix yours 🙂
https://github.com/Natizyskunk/vscode-sftp/releases
